### DISCUSSION TITLE
Cannot find Intel mode

### DISCUSSION THREAD

**Discussion by ankjxs** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1193] [Q&A]
AlDente 1.25.2 running on 15" 2014 macbook pro. Works perfectly in macOS, want it to run on windows using bootcamp. 
However, I cant find a toggle for intel mode. How can I enable this intel mode thing? I searched every menu of AlDente. Help

**Comment by MatthiasKerbl** [comment]
Hi @ankjxs,

On MacBook models from 2015 and prior, there is no need for the Intel Mode. Just set a charge limit in macOS and boot into Windows. Afterward, the charge limit will persist.

Best,
Matthias

**Reply by ankjxs** [reply]
I have set a charge limiter on macOS, but it is not carrying over to windows bootcamp.

**Reply by MatthiasKerbl** [reply]
Could you share screenshots of the settings you have in AlDente?

Best,
Matthias

**Reply by ankjxs** [reply]
<img width="1280" alt="Screen Shot 2024-03-17 at 8 58 34 PM" src="https://github.com/AppHouseKitchen/AlDente-Charge-Limiter/assets/163766725/99a379c0-0fd4-4d00-8692-b90c18e67139">
<img width="1280" alt="Screen Shot 2024-03-17 at 8 58 40 PM" src="https://github.com/AppHouseKitchen/AlDente-Charge-Limiter/assets/163766725/41e63ee7-a152-4fca-883f-a835c586981a">
<img width="1280" alt="Screen Shot 2024-03-17 at 8 58 48 PM" src="https://github.com/AppHouseKitchen/AlDente-Charge-Limiter/assets/163766725/7442b986-d551-4f31-b512-ba1e0f7fa3fd">
<img width="1280" alt="Screen Shot 2024-03-17 at 8 58 52 PM" src="https://github.com/AppHouseKitchen/AlDente-Charge-Limiter/assets/163766725/a609fe05-ccba-4106-b6fb-59ba235f6490">
<img width="1280" alt="Screen Shot 2024-03-17 at 8 58 58 PM" src="https://github.com/AppHouseKitchen/AlDente-Charge-Limiter/assets/163766725/d82e4498-51e8-472c-bf3d-d08cf2c00050">
<img width="1280" alt="Screen Shot 2024-03-17 at 8 59 03 PM" src="https://github.com/AppHouseKitchen/AlDente-Charge-Limiter/assets/163766725/15eea8ef-4002-4d1c-80b4-6daf49f4c6dc">
<img width="1280" alt="Screen Shot 2024-03-17 at 8 59 05 PM" src="https://github.com/AppHouseKitchen/AlDente-Charge-Limiter/assets/163766725/bcbbf926-27da-476b-9c46-ee101755ba02">

there you go. thank you for your time

**Reply by MatthiasKerbl** [reply]
Hi @ankjxs,

I am seeing just now that you are using AlDente free. To keep using the charge limit even in BootCamp you need AlDente Pro. You can try AlDente Pro for free through SetApp.
https://apphousekitchen.com/pricing/


Best,
Matthias
